Packs/DomainTools_Iris/Scripts/DomainToolsExtractAndEnrich/extract_and_enrich.yml

```
@@ -0,0 +1,333 @@
+args:
+- default: false
+  description: Resolve and enrich domains from this URL. Also accepts a comma-separated list of up to 6,000 URLs.
+  isArray: false
+  name: url
+  required: true
+  secret: false
+- default: false
+  description: Optionally include the investigate results into the Context Data. Defaults to false.
+  isArray: false
+  name: include_context
+  required: false
+  secret: false
+comment: Resolves a URL or fully qualified domain name (FQDN) and looks up a complete profile of the domain on the DomainTools Iris Enrich API.
+commonfields:
+  id: domainExtractAndEnrich
+  version: -1
+enabled: true
+name: domainExtractAndEnrich
```

Please follow the Pascal Case convention
```suggestion
  id: DomainExtractAndEnrich
  version: -1
enabled: true
name: DomainExtractAndEnrich
```

---

updated!